---
title: "BEAST2_Tutorial"
author: "Ernesto"
date: "2023-12-13"
output: html_document
---

## Purpose
Learn the workflow of BEAST2 to interpret the result of phylogenetic analysis


## Methods

The input data used comes from the sequence alignment of twelve primates
mitochondrial genome.

We used HKY substitution model with empirical frequencies, the clock model was 
kept default

The priors model was Calibrated Yule Model with a birthrate set to gamma with 
alpha in 0.001 and Beta in 1000

The MCMC settings was set the chain length to 1,000,000 and 10,000,000
to compare the effects

Tracer allowed us to visualize the data in different ways 

Tree Creation help us create the tree by annotating wit Tree annotator and 
modifying the aspect of it and fonts with FigTree

## Results

The results was a phylogenetic tree with the data 

![](Tracer.png)
![](Tree.png)

## References

Bouckaert, Remco, Timothy G. Vaughan, Joëlle Barido-Sottani, Sebastián Duchêne, Mathieu Fourment, Alexandra Gavryushkina, Joseph Heled, et al. 2019. “BEAST 2.5: An Advanced Software Platform for Bayesian Evolutionary Analysis.” PLOS Computational Biology 15 (4): e1006650..

Joëlle Barido-Sottani, Veronika Bošková, Louis du Plessis, Denise Kühnert, Carsten Magnus, Venelin Mitov, Nicola F. Müller, Jūlija Pečerska, David A. Rasmussen, Chi Zhang, Alexei J. Drummond, Tracy A. Heath, Oliver G. Pybus, Timothy G. Vaughan, Tanja Stadler (2018). Taming the BEAST – A community teaching material resource for BEAST 2. Systematic Biology, 67(1), 170–-174. doi: 10.1093/sysbio/syx060